src/component/common/Header/MobilePanel/index.tsx

```diff
@@ -98,26 +131,8 @@ export default function MobilePanel() {
               </ul>
             </div>
 
-            {CATEGORY.map((categoryInfo) => (
-              <div key={categoryInfo.title}>
-                <div>
-                  <div className={styles['mobile-header__category-title']}>
-                    {categoryInfo.title}
-                  </div>
-                  <ul className={styles['mobile-header__sub-menus']}>
-                    {categoryInfo.submenu.map((subMenu) => (
-                      <li
-                        className={styles['mobile-header__sub-menu']}
-                        key={subMenu.title}
-                      >
-                        <Link to={subMenu.link}>
-                          {subMenu.title === '가게정보' && subMenu.title}
-                        </Link>
-                      </li>
-                    ))}
-                  </ul>
-                </div>
-              </div>
+            {targetCategory.map((category: HeaderCategory) => (
+              <PanelContent hideSidebar={hideSidebar} category={category} />
```

Comment:
지금 unique한 key가 없어서 에러가 발생하고있어요
다음 예시처럼 키값 추가가 필요해 보여요.
```suggestion
            {targetCategory.map((category: HeaderCategory) => (
              <PanelContent hideSidebar={hideSidebar} category={category} key={category.title}/>
```

Reply:
수정하였습니다 b8b0d9bb7c0b3a058f2ce57f05bf08151253c669